### ISSUE TITLE
Not able to "hide_tabs" by title if title contains spaces

### ISSUE THREAD

**Issue by finity69x2** [maykar/custom-header#262]
trying to set "hide_tabs" for tab by title doesn't work if the tab title contains spaces in it.

code example that won't work:

lovelace_views/lovelace_garage_view.yaml

```
title: Garage Power
cards:
  - type: entities
    title: Garage Power strip
    show_header_toggle: false
    state_color: true
    entities:
.
.
.
```

ui-lovelace.yaml

```
custom_header:
  compact_mode: true
  hide_config: true
  hide_raw: true
  hide_help: true
  voice_hide: true
  chevrons: false
  hide_tabs: >-
    {{ 'Alerts, ' if states.sensor.nws_alerts_are_active.state == "off" and states.input_boolean.show_nws_test_alerts_view.state == "off" else '' }}
    {{ 'Tank, ' if states.input_boolean.show_tank_view.state == "off" else '' }}
    {{ 'Garage Power, ' if states.input_boolean.show_garage_power_view.state == "off" else '' }}
.
.
.
views:
  - !include '/config/lovelace_views/lovelace_garage_view.yaml'
```

in the example the "Garage Power" view won't hide.

**Comment by maykar** [comment]
Thank you, forgot about this one. It was reported on the forums, but my memory isn't what it used to be. A workaround is setting a path for the view in Lovelace and then use that in hide tabs or use the tabs index.

Will get this fixed in next release.

**Comment by maykar** [comment]
This has been addressed in the latest pre-release 1.7.8. Please, report back here and let me know if it has fixed your issues.

To view pre-releases in HACS go to the Custom Header listing under "Frontend", click the 3 dots in the right hand corner, select "Reinstall", turn on "Show beta versions", select 1.7.8, and hit "Install". Clear cache afterwards just in case.

If you do not see the version 1.7.8 reported in your browser's dev tools console (F12). You may need to reinstall or clear cache. If you installed manually, make sure you're adding to the URL version number in your Lovelace resources.

**Comment by maykar** [comment]
Have tested this in a few different ways and am confident that it has been resolved. Closing as I'm pushing this into a full release. If you're still having issues, please let me know.
